TinyConstraints/Classes/Constraints.swift

```diff
@@ -55,7 +55,7 @@ public extension Collection where Iterator.Element == Constraint {
 }
 
 #if os(OSX)
-public extension Constraint {
+@MainActor public extension Constraint {
```

Comment:
Not needed since `Constraint` is already isolated to the `@MainActor`

```suggestion
public extension Constraint {
```